lib/mailer_previews/admin_mailer_preview.rb

```diff
@@ -0,0 +1,16 @@
+class AdminMailerPreview < ActionMailer::Preview
+  
+  def contributor_stats
+    AdminMailer.contributor_stats(63, 2.weeks.ago, Time.now, 'trishablewis@gmail.com')
```

Comment:
Change the email to the admin mail address (defined in the initializer, I think.)